lms/static/sass/shared/_course_object.scss

```diff
@@ -180,78 +191,116 @@
         }
       // }
 
-      .info {
-        background: $content-wrapper-bg;
-        height: 220px + 130px;
-        left: 0px;
-        position: absolute;
-        top: 0px;
-        @include transition(all 0.15s linear 0s);
-        width: 100%;
-        overflow: hidden;
-
+      .course-image {
         .cover-image {
-          height: 200px;
+          height: 160px;
           overflow: hidden;
-          width: 100%;
+
+          &:before {
+            @include fade-transition();
+            content: '';
+            position: absolute;
+            top: 0;
+            left: 0;
+            background: black;
+            opacity: 0;
+            width: 100%;
+            height: 160px;
+            z-index: 10;
+          }
+
+          &:after {
+            @include fade-transition();
+            @include box-sizing(border-box);
+            @include calc( top, '80px - 25px');
+            @include calc( left, '50% - 100px');
+            content: 'LEARN MORE';
+            position: absolute;
+            width: 200px;
+            height: 50px;
+            z-index: 20;
+            border: 3px solid white;
+            border-radius: 3px;
+            font-family: $sans-serif;
+            font-weight: $font-semibold;
+            font-size: 1.1em;
+            line-height: 44px;
+            color: white;
+            padding: 0 20px;
+            text-align: center;
+            opacity: 0;
+          }
 
           img {
-            display: block;
-            min-height: 100%;
             width: 100%;
```

Comment:
The idea here was to maintain the image ratio while constraining it to the width of it's container, because we want to see the full width of the course image the whole time.

Setting `display: block` rule instead didn't allow that. See the image bellow:
![screen shot 2015-04-10 at 10 32 31](https://cloud.githubusercontent.com/assets/6833568/7084465/fb5dc990-df6d-11e4-8112-6a0cce8904cb.png)
Image wasn't presented in the full width.